src/langchain_google_alloydb_pg/async_vectorstore.py

```diff
@@ -217,15 +217,15 @@ async def create(
     def embeddings(self) -> Embeddings:
         return self.embedding_service
 
-    async def __aadd_embeddings(
+    async def aadd_embeddings(
```

Comment:
Let's keep this private here since it is the pure implementation of the vector store that we don't expect users to use directly